buildscripts/kokoro/linux_artifacts.cfg

```diff
@@ -6,6 +6,7 @@ timeout_mins: 60
 
 action {
   define_artifacts {
+    regex: "github/grpc-java/**/build/test-results/**/sponge_log.xml"
```

Comment:
Same as my comment for the other 2: can we retain the html files for easy readability?